static/sass/_sidebar.scss

```diff
@@ -1,3 +1,5 @@
+@use 'sass:math';
+
 .docs-container {
```

Comment:
Something to verify - is this needed anywhere on the site?

It seems the docs section (https://microk8s.io/docs) has been using official Vanilla docs layout, so this styling maybe is not used anywhere anymore?